CHANGELOG.md

```diff
@@ -50,6 +50,10 @@ and this project adheres to
   limit to any those constructors ([#961]).
 - cosmwasm-std: Change `Event`'s constructor - it no longer takes a vector of
   attributes and instead constructs an empty one
+- cosmwasm-std: Rename `SubcallResponse` to `SubMsgExecutionResponse`.
+- contracts: Rename `ReflectSubCall` to `ReflectSubMsg` and `SubCallResult` to
+  `SubCallMsg` in the `reflect` contract.
+- cosmwasm-std: Rename the `subcall` module to `submessages`.
```

Comment:
I usually do not list changes that are completely internal and do not affect users. But it does not harm as well.